CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+Contributing to the OCaml opam repository
+-----------------------------------------
+
+Contributions under the form of new packages, issue reports and pull
+requests to fix and enhance the quality of packages are always
+welcome. Thanks for your time and involvement.
+
+The following lists a few points that help in making the contribution
+process and repository management as smooth and efficient as possible
+for everyone involved.
+
+Note that if your have an issue with the opam tool itself this
+should not be filed here but on the opam tool
+[issue tracker](https://github.com/ocaml/opam/issues).
+
+
+Reporting package installation issues
+-------------------------------------
+
+To report package installation issues:
+
+1. First try to search the issue tracker to see if someone already
+   reported the issue. If that is the case mention on this issue that
+   you are also affected and, if available, add more diagnostic details.
+2. If you create a new issue make sure the issue title mentions the
+   package name and the affected version.
+2. Try to give as much information as possible about your setup. In
+   particular, on installation failures, add the the exact opam
+   install invocation and the resulting tool output to the issue body.
```

Comment:
even if it the tool responsibility should do it, I think it is worth mentioning `opam config report` here.